server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/json/EmailGetSerializer.scala

```diff
@@ -57,10 +57,17 @@ object EmailGetSerializer {
   private implicit val rawHeaderWrites: Writes[RawHeaderValue] = Json.valueWrites[RawHeaderValue]
   private implicit val textHeaderWrites: Writes[TextHeaderValue] = Json.valueWrites[TextHeaderValue]
   private implicit val addressesHeaderWrites: Writes[AddressesHeaderValue] = Json.valueWrites[AddressesHeaderValue]
+  private implicit val GroupNameWrites: Writes[GroupName] = Json.valueWrites[GroupName]
+  private implicit val emailAddressGroupWrites: Writes[EmailAddressGroup] = (o: EmailAddressGroup) =>
```

Comment:
Cant we just write `= Json.writes[EmailAddressGroup]`

Reply:
i want to retain the groupName = null for addresses without a group (as spec says), therefore this approach